samples/streamlit-in-snowflake/sis-deploy/snowflake.yml

```diff
@@ -0,0 +1,9 @@
+definition_version: 1
+streamlit:
+  name: sample_app
+  query_warehouse: my_streamlit_warehouse # Note that an XS warehouse is recommended
+  main_file: streamlit_app.py
+  env_file: environment.yml
+  pages_dir: pages/
+  additional_source_files:
```

Comment:
maybe we can add a comment that is like "this grabs all the python files in the common folder"